coral/src/app/features/browse-topics/components/search/SearchTopics.tsx

```diff
@@ -28,10 +23,14 @@ function SearchTopics(props: SearchTopicsProps) {
       <label className={"visually-hidden"} htmlFor={"topics-search"}>
         Search topics
       </label>
-      <Flexbox>
+      <Grid
+        cols={"2"}
+        rows={"1"}
+        style={{ gridTemplateColumns: "minmax(auto, 300px) auto" }}
+      >
         <InputBase
           type={"search"}
-          placeholder="Topic name"
+          placeholder="Search for…"
           value={searchTerm}
           onChange={(event) => setSearchTerm(event.target.value)}
```

Comment:
Nitpick / Not related to the issue solved by this PR: I notice that we maintain two sets of states for `[searchTerm, setSearchTerm]`, in this component `SearchTopics` and in the parent `BrowseTopics`. 

Maybe only the one in the parent is necessary? We could change the parameter to be `event: React.ChangeEvent<HTMLInputElement>` add these lines to the `searchTopics` function we pass as the `onChange` prop: 
```ts
    event.preventDefault();
    const trimmed = searchTerm.trim();
```

And then we could remove the state here and have

```ts
    onChange={onChange}
```

We might even rename the function and the `SearchTopics` prop as `handleSearchChange` to get some extra clarity.